File: Data Structures & Algorithms/maximum-ascending-subarray-sum/submission-2.py
class Solution:
    def maxAscendingSum(self, nums: List[int]) -> int:
        currSum = nums[0]
        maxSum = nums[0]

        for i in range(1,len(nums) ) :
            if nums[i] > nums[i-1] :
                currSum += nums[i]
            else :
                currSum = nums[i]
            
            maxSum = max(maxSum, currSum)

        return maxSum

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution {
    public int longestConsecutive(int[] nums) {
       HashSet<Integer> numSet = new HashSet<>();
       for(int i = 0 ; i < nums.length ; i++) 
       {
        numSet.add(nums[i]);
       };

       int streak = 0;

       for(int i = 0 ; i < nums.length ; i++)
       {

        int currentNum = nums[i];

        int currentStreak = 0;
        while(numSet.contains(currentNum))
        {
            currentStreak++;
            currentNum = currentNum+1;
            streak = Math.max(currentStreak,streak);
        }

       };

       return streak;
    }
}
